pkg/chunked/internal/path.go

```diff
@@ -0,0 +1,12 @@
+package internal
+
+import (
+	"path/filepath"
+)
+
+// CleanAbsPath removes any ".." and "." from the path
+// and ensures it starts with a "/".  If the path refers to the root
+// directory, it returns "/".
+func CleanAbsPath(path string) string {
```

Comment:
`pkg/chunked/internal` is included in all c/image callers (incl. ones that don’t use the c/storage transport, e.g. when they only want to use the registry client), compare the comment in `pkg/chunked/compression.go`. Please use some other subpackage.

Reply:
I assumed it was an acceptable cost.  Moved to `storage/internal`

Reply:
I agree that the cost of this one function is trivial.

Breaking the precedent, and opening the door to adding more compression-unrelated functionality or dependencies into that subpackage, would be a long-term problem.

Arguably, with the addition of `pkg/chunked/internal/*` subpackages, this special property of `pkg/chunked/internal` itself is becoming less clear. Shall I file a PR to move that code into `pkg/chunked/internal/minimal` or something like that?

Reply:
> Arguably, with the addition of `pkg/chunked/internal/*` subpackages, this special property of `pkg/chunked/internal` itself is becoming less clear. Shall I file a PR to move that code into `pkg/chunked/internal/minimal` or something like that?

yes thanks, that would be better than what we have now

Reply:
> Shall I file a PR to move that code into `pkg/chunked/internal/minimal` or something like that?

https://github.com/containers/storage/pull/2196 .